### ISSUE TITLE
CHARLATANO AND RATPOISON ARE the same?

### ISSUE THREAD

**Issue by tilenzz** [Jire/Charlatano#906]
I KNEW ABOUT A RATPOISON, HE HAS THE SAME SECURITY OF THE CHARLATAN ?? BECAUSE HIM AND PRACTICALLY A CHARLATAN WITH LESS

**Comment by Bobby8904** [comment]
Its the same thing Charlatano is the simple old base of the cake where ratposin is the cake, icing, and cherry. (Same dev)

**Comment by Zyphix** [comment]
@tilenzz Yes Rat Poison is just a newer and improved version with the same developer. I’m not sure if Jire just collaborated with another dev but Jire doesn’t check up on this same cheat forum as much as he used to and is seen more in Ratpoison x Charlatano cheat forum
